src/blocks/events/event-item/test/save.spec.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import '@testing-library/jest-dom/extend-expect';
+import { registerBlockType, createBlock, serialize } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies.
+ */
+import { name, settings } from '../index';
+
+// Make variables accessible for all tests.
+let block;
+let serializedBlock;
+
+const DEFAULT_ATTRIBUTES = {
+	title: "Some title",
+	description: "Some description",
+	eventDay: "25",
+	eventMonth: "Nov",
+	eventYear: "2099",
+	eventTime: "5:25",
+	eventLocation: "Somewhere",
+}
+
+describe( name, () => {
+	beforeAll( () => {
+		// Register the block.
+		registerBlockType( name, { category: 'common', ...settings } );
+	} );
+
+	beforeEach( () => {
+		// Create the block with the minimum attributes.
+		block = createBlock( name );
+
+		// Reset the reused variables.
+		serializedBlock = '';
```

Comment:
`serializedBlock` could be its own variable in each test, as we discussed the other day, instead of doing this.